src/scaffoldmaker/meshtypes/meshtype_3d_nerve1.py

```diff
@@ -0,0 +1,1982 @@
+                                                     (1, Node.VALUE_LABEL_D2_DS1DS2, si + 4),
+                                                     (1, Node.VALUE_LABEL_D_DS3, si + 5),
+                                                     (1, Node.VALUE_LABEL_D2_DS1DS3, si + 6),
+                                                     (3, Node.VALUE_LABEL_VALUE, si + 7),
+                                                     (3, Node.VALUE_LABEL_D_DS1, si + 8),
+                                                     (3, Node.VALUE_LABEL_D_DS2, si + 9),
+                                                     (3, Node.VALUE_LABEL_D2_DS1DS2, si + 10),
+                                                     (3, Node.VALUE_LABEL_D_DS3, si + 11),
+                                                     (3, Node.VALUE_LABEL_D2_DS1DS3, si + 12)])
+                    si += 12
+            ln = 2
+            for n2 in range(2):
+                valueExpression = [(Node.VALUE_LABEL_VALUE, [])]
+                d_ds1Expression = [(Node.VALUE_LABEL_D_DS1, [])]
+                d_ds2Expression = []
+                pole = (e + n2) % 4
+                if pole == 0:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS2, [2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS2, [2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS3, [3]))
+                elif pole == 1:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS3, [2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS3, [2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS2, [1, 3]))
+                elif pole == 2:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS2, [1, 2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS2, [1, 2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS3, [1, 3]))
+                elif pole == 3:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS3, [1, 2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS3, [1, 2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS2, [3]))
+
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_VALUE, valueExpression)
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_D_DS1, d_ds1Expression)
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_D_DS2, d_ds2Expression)
+                ln += 2
+
+            remapEftLocalNodes(eft2dNV, 3, [1, 2, 3, 2])
+            facetemplate_branch_root = mesh2d.createElementtemplate()
+            facetemplate_branch_root.setElementShapeType(Element.SHAPE_TYPE_SQUARE)
+            facetemplate_branch_root.defineField(coordinates, -1, eft2dNV)
+            facetemplate_and_eft_list_branch_root[e] = (facetemplate_branch_root, eft2dNV)
+
+        # load data from file
+        print('Extracting data...')
```

Comment:
I think these print statements and others tracking the progress of the algorithm should go.